docs/Options/Changelog.rst

```diff
@@ -0,0 +1,13 @@
+
```

Comment:
We also use the wiki for Changelog right now in deep, May be should talk about moving the existing changelog here?
@AdityaKhatri cc: @nayid-DFS